packages/bpk-component-autosuggest/src/BpkAutosuggestSuggestion.tsx

```diff
@@ -16,31 +16,34 @@
  * limitations under the License.
  */
 
-/* @flow strict */
-
-import PropTypes from 'prop-types';
-import type { Node } from 'react';
+import type { ReactNode, FunctionComponent, SVGProps } from 'react';
 
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
 import { cssModules } from '../../bpk-react-utils';
 
 import STYLES from './BpkAutosuggest.module.scss';
 
 const getClassName = cssModules(STYLES);
 
 type Props = {
-  value: Node,
-  indent: boolean,
-  className: ?string,
-  icon: Function,
-  subHeading: ?Node,
-  tertiaryLabel: ?string,
+  value: ReactNode;
+  subHeading?: ReactNode;
+  tertiaryLabel?: string;
+  icon?: FunctionComponent<SVGProps<SVGSVGElement>>;
+  indent?: boolean;
+  className?: string;
 };
 
-const BpkSuggestion = (props: Props) => {
+const BpkSuggestion: FunctionComponent<Props> = ({
+  className,
```

Comment:
also noticed default values are missing, is that intentional?

Reply:
Yea its intentional. In the js file they are all nulls and false so passing the default values has no purpose.

```
BpkSuggestion.defaultProps = {
  subHeading: null,
  tertiaryLabel: null,
  icon: null,
  indent: false,
  className: null,
};
```
With typescript for the optional props setting defaultProps as non values and not passing any values equates to the same thing.

I think we would only need this if the default prop had a value, like we're doing in BpkAutosuggest with theme